### DISCUSSION TITLE
what about Svelte integration?

### DISCUSSION THREAD

**Discussion by pabl-o-ce** [apollographql/apollo-server#7795] [Ideas]
Hi, guys I was looking at your issues that someone create a [issue](https://github.com/apollographql/apollo-server/issues/5897) to integrate more with svelte and in the comments you said something about Apollo would be more easy to integrate...

Any guide how integrate with svelte?

Thanks for your work.

**Comment by trevor-scheer** [comment]
I'm completely unfamiliar with svelte/kit, but we do have a general [guide on creating integrations](https://www.apollographql.com/docs/apollo-server/integrations/building-integrations) in the docs.

All of the existing integrations of AS4 (that I'm aware of) currently exist in the [`apollo-server-integrations` GH org](https://github.com/apollo-server-integrations) if you'd like to see some examples. If you're interested in building and maintaining an integration for public use, I'd be happy to add your project to the org and get you set up to publish under the `@as-integrations` npm scope.
